includes/CSSUtilities.php

```diff
@@ -131,61 +139,35 @@ private function get_inline_css() {
 		
 		return $css;
 	}
-	
-	/**
-	 * Get the URL for the asset, checking if the remote URL is valid.
-	 *
-	 * @param string $remote_url The remote URL for the asset.
-	 * @param string $fallback_url The fallback URL if the remote asset is invalid.
-	 * @return string The valid URL for the asset.
-	 */
-	private function get_asset_url( string $remote_url, string $fallback_url ) : string {
-		return $this->is_valid_remote_file( $remote_url ) ? $remote_url : $fallback_url;
-	}
 
-	/**
-	 * Get the version number for the asset, either from remote or fallback.
-	 *
-	 * @param string $remote_url The remote URL for the asset.
-	 * @return int|string The version number.
-	 */
-	private function get_asset_version( string $remote_url ) {
-		return $this->is_valid_remote_file( $remote_url ) ? $this->get_remote_assets_version() : NFD_WONDER_BLOCKS_VERSION;
-	}
-	
 	/**
 	 * Check if a remote file is valid.
 	 *
-	 * @param string $url URL of the remote file.
+	 * Stores the resulting HTTP status (or "error") in a transient for 24 hours.
+	 * {@see wp_remote_retrieve_response_code()} returns 200 even for redirects.
 	 *
-	 * @return bool
+	 * @param string $url URL of the remote file.
 	 */
 	private function is_valid_remote_file( string $url ): bool {
-		$response = \wp_remote_get( $url, array( 'timeout' => 5 ) );
-		if ( is_wp_error( $response ) ) {
-			return false;
-		}
+		// Reverse the url because transient key length is limited and truncated and the unique part of the URL is its end.
```

Comment:
Should we use md5 instead of strrev for generating the transient key? md5 provides a fixed-length, unique hash that fits well within transient key length limits.

`$transient_key = 'nfd_css_utilities_valid_' . md5($url);`